stories/13-wordpress-components/template-part.stories.mdx

```diff
@@ -0,0 +1,61 @@
+<Meta title="WordPress Components|Template Part" />
+
+# Template Part
+Load a template part.
+
+Renders the `Fragment` styled component [(docs)](?path=/docs/styled-components-fragment--basic-eample).
+
+[View Source](https://github.com/alleyinteractive/wp-irving/tree/master/inc/components/components/template-part).
+
+## Overview
+This component is a wrapper for `get_template_part()` [(docs)](https://developer.wordpress.org/reference/functions/get_template_part/). This component allows for dynamically loading template parts.
+
+During the configuration callback, this component will add `${slug}.json` and `${slug}-${name}.json` to the `templates` array. The children callback will iterate through `templates`, setting the component `children` with the children of the first successful hydration.
```

Comment:
I noticed in the ticket the note: `Note: Should/can this eventually replace the template-parts/$slug functionality?`. It may be worth addressing that here to avoid confusion about what the recommended usage is for `irving/template-part` vs `template-parts/$slug`. Even if it's still being sorted out, that would be helpful to know.